Library/Homebrew/cmd/link.rb

```diff
@@ -24,15 +24,17 @@ def link
     mode.dry_run = true if ARGV.dry_run?
 
     ARGV.kegs.each do |keg|
-      if HOMEBREW_PREFIX.to_s == "/usr/local" && keg.name == "openssl"
-        opoo <<-EOS.undent
-          Refusing to link: openssl
-          Linking keg-only OpenSSL means you may end up linking against the insecure,
-          deprecated system version while using the headers from the Homebrew version.
-          Instead, pass the full include/library paths to your compiler e.g.:
-            -I#{HOMEBREW_PREFIX}/opt/openssl/include -L#{HOMEBREW_PREFIX}/opt/openssl/lib
-        EOS
-        next
+      if HOMEBREW_PREFIX.to_s == "/usr/local"
+        if keg.name.start_with?("openssl") || keg.name.start_with?("libressl")
```

Comment:
We're making progress on that front though.

2356 formulae currently have some form of `cellar :any`, another 451 are `bottle :unneeded`, another 5 are `bottle :disable`. So that leaves... 783 non-portable bottles. Rough numbers, obviously, but we're making solid progress here.

We'll get a lot more wrapped up once we work out how to relocate NLS stuff as well.